tasks/libs/notify/utils.py

```diff
@@ -43,3 +44,28 @@ def get_ci_visibility_job_url(
         extra_args = ''.join([f'&{key}={value}' for key, value in extra_args.items()])
 
     return CI_VISIBILITY_JOB_URL.format(name=name, extra_flags=extra_flags, extra_args=extra_args)
+
+
+def should_notify():
+    from tasks.libs.ciproviders.gitlab_api import get_pipeline
+
+    CONDUCTOR_ID = 8278
+    # agent = get_gitlab_repo()
+    # pipeline = agent.pipelines.get(os.environ['CI_PIPELINE_ID'])
+    pipeline = get_pipeline(PROJECT_NAME, os.environ['CI_PIPELINE_ID'])
+    if (
+        os.environ['CI_PIPELINE_SOURCE'] != 'pipeline'
+        or os.environ['CI_PIPELINE_SOURCE'] == 'pipeline'
+        and pipeline.user['id'] == CONDUCTOR_ID
+    ):
+        return True
+    return False
```

Comment:
```suggestion
   return (
        os.environ['CI_PIPELINE_SOURCE'] != 'pipeline'
        or os.environ['CI_PIPELINE_SOURCE'] == 'pipeline'
        and pipeline.user['id'] == CONDUCTOR_ID
    )
```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏This would be shorter and still readable imo, WDYT?

Reply:
Agreed, thanks!